portfolio/src/main/webapp/index.html

```diff
@@ -57,31 +57,44 @@ <h2>My Favourite Place</h2>
 
       <h2>My Favourite Food</h2>
       <p>
-        My favourite food is any brunch food because there are no rules at brunch, anything can be a brunch food if it tries. 
+        My favourite food is any brunch food because there are no rules at brunch, anything can be a brunch food if it tries.
         I love taking pictures of food, here are some of my brunch photos.
       </p>
       <div class="food-slideshow">
         <img src="/images/food/food0.jpg">
         <button class="prev-slide-button">Previous</button>
         <button class="next-slide-button">Next</button>
       </div>
-    
+
       <h2 id="comment-section">Comments</h2>
+
+      <form onsubmit="loadComments()">
+        <label for="limit">Show Comments:</label>
+        <select name="limit">
+          <option value="5">5</option>
+          <option value="10">10</option>
+          <option value="20">20</option>
+          <option value="50">50</option>
+        </select>
+        <input type="submit"/>
+      </form>
       <div id="comment-container"></div>
 
       <form action="/data" method="POST">
```

Comment:
Notice how when you set your comment limit to something that is not the default and then you submit a new comment, it goes back to the default limit. This is because your forcing the user to go to the /data page and then the servlet forces them back. You can instead use a POST fetch.